README.md

```diff
@@ -1,7 +1,62 @@
 # CVCoach
 
+This repository is a project for METCS673. This project focuses on using AI to automate and improve the process of resume evaluation and interview preparation. By incorporating Retrieval-Augmented Generation (RAG), we ensure that our application can provide more accurate, context-relevant reviews.
+
 An artifical intelligence tool powered by the ???. It can provide several services for job seekers, interviewers and HRs, such as refining the CV, mocking interviews and writing questions based on resumes.
 
+## Data Collection and preprocessing
```

Comment:
The details of implementation should be moved into a separated file, like the `doc/development_manual.md` did. But you can do it later.

Reply:
Sure. I will do it later.